### DISCUSSION TITLE
Download default source strings over-the-air

### DISCUSSION THREAD

**Discussion by jablkojem** [crowdin/mobile-sdk-android#196] [General]
Hello, is it possible to somehow download source strings.xml when using over-the-air content delivery? I have spanish translations in my crowdin project and if set spanish in system settings, I can see that 'es' translations are downloaded and user can see new strings. I would like to affect also default strings.xml, which is used if app is not supporting system language - lets say user have german language in system settings - app is supporting just english and spanish - in this case default strings.xml is used. I am unable to change this default strings.xml over-the-air, is it possible to do this somehow?

**Comment by ira-sem** [comment]
Hi @jablkojem !
Could you please let us know what do you mean by "change the default strings.xml"? When there is no translation for the chosen language, the system by default shows the English source. Do you expect to see something else?

**Reply by jablkojem** [reply]
Default strings.xml - I mean source file for translations. I tested it now, and the behavior is that if I set for example german as a system language (app is supporting english and spanish - these languages are also on crowdin) - app is using default strings.xml (source strings), not english version

**Comment by ira-sem** [comment]
@jablkojem yes, this is expected by default
and your default (source) language is not English as far as I understand?

**Reply by jablkojem** [reply]
Our source language is english, but there is a difference - if user set english as a system language, strings are taken from values-en/strings.xml - I can see in logs that crowdin is downloading this strings file 
`StringDataRemoteRepository. Loading string data from /project/app/src/main/res/values-en/strings.xml`
so if user set english as a system language, we can change strings and it works over-the-air. But if user has for example german language, strings are not taken from values-en/strings.xml but from values/strings.xml - and this strings.xml file is not downloaded from crowdin, so we are unable to change strings over-the-air for this type of user.

**Reply by jablkojem** [reply]
So my question is, is it possible to somehow change source strings.xml over-the-air? Or is this functionality aimed at translations only?

**Reply by ira-sem** [reply]
just checked with the team and this is on our roadmap. We will let you know once we have any updates from the tech team on when it will be ready to do as you expect

**Reply by jablkojem** [reply]
perfect, thank you! 👍

**Reply by ira-sem** [reply]
Hi there! 
Just wanted to let you know that we improved the functionality for OTA Content Delivery and added the possibility to export the source file (source language) by transferring the source language code in the same way as it is with the target language

That is, you can use the link below to specify the source language instead of the target language and receive the source file
https://distributions.crowdin.net/{distribution_hash}/content/{path_to_file}?timestamp={timestamp_from_manifest}
